miio/chuangmi_ir.py

```diff
@@ -13,7 +13,7 @@ class ChuangmiIrException(DeviceException):
 class ChuangmiIr(Device):
     """Main class representing Chuangmi IR Remote Controller."""
 
-    def learn(self, key: int):
+    def learn(self, key: int = 1):
```

Comment:
You have here whitespaces around `=` while none for frequency in play, looks fine otherwise so ship it!